docs/features/tls_security_profile.adoc

```diff
@@ -0,0 +1,49 @@
+== https://issues.redhat.com/browse/LOG-3270[LOG-3270]: OCP Cluster-wide Cryptographic Policies and Output Crypto Configuration
+
+This feature makes collector deployments comply with the OCP cryptographic policy that is defined for
+the https://docs.openshift.com/container-platform/4.6/rest_api/config_apis/apiserver-config-openshift-io-v1.html[cluster] config settings.
+Additionally, this feature enables the capability for administrators to define a TLS profile to be used when forwarding
+logs to different outputs.
+
+TLS profiles are uniquely configurable for each defined output or by defining a single profile to apply to all log forwarding outputs. Profiles
```

Comment:
TLS profiles are ~~uniquely~~ configurable for each ~~defined~~ output separately